Drop-In/src/main/java/com/braintreepayments/api/DropInState.java

```diff
@@ -0,0 +1,6 @@
+package com.braintreepayments.api;
+
+enum DropInState {
+    IDLE,
```

Comment:
What does idle mean here? Is this active?

Reply:
Yeah we could change the name of this particular enum case. I mainly needed an initial state to differentiate from `FINISHING` so that any observers listening to the live date property for `DropInState` would know not to take action.